src/app/components/Curation/index.tsx

```diff
@@ -68,7 +69,13 @@ const Curation = ({
         />
       );
     case MOST_READ:
-      return <MostRead data={mostRead} columnLayout="twoColumn" />;
+      return (
+        <MostRead
+          data={mostRead}
+          columnLayout="twoColumn"
+          headingBackgroundColour={GHOST}
+        />
```

Comment:
NOTE: `headingBackgroundColour` has been passed into the MostRead component being called from Curation here due to a small bug I noticed where the background colour surrounding the "Most Read" heading was a slightly different colour to the background on the live Homepage and therefore opted to implement a fix as part of this PR.